src/Account.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tighten\SolanaPhpSdk;
+
+use Tighten\SolanaPhpSdk\Util\Ed25519Keypair;
+
+class Account
+{
+    protected KeyPair $keyPair;
+
+    public function __construct($secretKey = null)
+    {
+        if ($secretKey) {
+            $secretKeyString = is_string($secretKey)
+                ? $secretKey
+                : Ed25519Keypair::array2bin($secretKey);
+
+            $this->keyPair = KeyPair::fromSecretKey($secretKeyString);
+        } else {
+            $this->keyPair = new KeyPair();
+        }
+    }
+
+    /**
```

Comment:
I usually drop the doc blocks when we're using return types. Any reason to keep them, though?

Reply:
I know this is pretty debated, but I LOVE the doc blocks for the visual separation, even if their contents add no value. I would enjoy leaving them in :)

Reply:
:) Fine -- let's just make sure they are kept up to date please.